services/workflows-service/src/workflow/workflow.service.ts

```diff
@@ -903,6 +908,10 @@ export class WorkflowService {
         documents: context?.documents?.map(
           (document: DefaultContextSchema['documents'][number]) => ({
             ...document,
+            decision: {
+              ...document?.decision,
+              status: document?.decision?.status === null ? undefined : document?.decision?.status,
```

Comment:
The ability to cancel a request for re-upload introduced `status: null`. `null` is needed due to the nature of the `merge` method we use but `null` also breaks a number of places (validation).